internal/publish/connect_cloud/content_request_base.go

```diff
@@ -0,0 +1,70 @@
+package connect_cloud
+
+// Copyright (C) 2025 by Posit Software, PBC.
+
+import (
+	"fmt"
+
+	"github.com/posit-dev/publisher/internal/clients/types"
+	"github.com/posit-dev/publisher/internal/config"
+)
+
+func getCloudContentType(contentType config.ContentType) (types.ContentType, error) {
```

Comment:
I have suspicions that this isn't quite correct/will need some tweaking, but I'm content to leave that for later.

Reply:
Is there an issue to confirm this later?